lib/locales/urhobo/names/last_name.js

```diff
@@ -0,0 +1,19 @@
+module.exports = [
+  'Edewor ',
+  'Adaware',
+  'Okonedo',
+  'Ejomafuvwe ',
+  'Odibo',
+  'Onanefe',
+  'Ofuako',
+  'Okpara',
+  'Orodena',
+  'Okoro',
+  'Umukoro',
+  'Iniovosa',
+  'Odafe',
+  'Orogun',
+  'Sapele',
+  'Ododo',
+];
+  
```

Comment:
You might want to lose the trailing spaces on this line, that's what's causing this red warning here.